backend/api/endpoints/misc.py

```diff
@@ -24,14 +24,13 @@ def genQR(teamCode: str):
     # Other cases do not interest us.
     teamCode = teamCode.replace(" ", "%20")
 
-    img = qrcode.make('https://shifra.klubfitpp.cz/join/' + teamCode, image_factory = qrcode.image.svg.SvgImage)
+    img = qrcode.make('https://shifra.klubfitpp.cz/team/join/code?code=' + teamCode, image_factory=qrcode.image.svg.SvgImage)
 
     stream = io.BytesIO()
     img.save(stream)
-    stream.seek(0)
 
-    # Save the image into fastapi response stream
-    return StreamingResponse(stream, media_type="image/svg")
+    image = stream.getvalue()
+    return base64.b64encode(image)
```

Comment:
Tohle jsme se soptíkem vyřešily, prosím roll back. Šlo o špatný content-type

Reply:
Ok, po pullu to nechám být